### ISSUE TITLE
Display execution time.

### ISSUE THREAD

**Issue by tzzed** [genjidb/genji#443]
### Proposal
Currently there is no way to know the execution's time of a query.
As Sqlite with a command `.timer on|off`. 

```shell
genji>.timer on
genji>select * from foo;
{
  "Username": "Tonydorrf"
}
Time: 0ms
genji>
```
Then turn off the timer
```shell
genij>.timer off
genji>select * from foo;
{
  "Username": "Tonydorrf"
}
genji>
```

### Motivation
Sometimes users need to measure performances of requests, index efficiency....
Sqlite proposes the command `.timer on|off` which measures the time of every command while the timer is active `.timer on` and it doesn't after turning off `.timer off`.
The timer is off by default.

**Comment by asdine** [comment]
Good idea! Interested?

**Comment by tzzed** [comment]
Yes!

**Comment by asdine** [comment]
Closed by #445
